pset-iterators.js

```diff
@@ -184,6 +227,22 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+// 1. determine years in office by returning year left office minus year took offiuce
+// 2. convert the strings to a varioable
+
+const notCurrentInOffice = presidents.filter((president) => {
```

Comment:
@kcycle nice work, using another separate `.filter()` to exclude the current president